hail/src/main/scala/is/hail/backend/local/LocalBackend.scala

```diff
@@ -62,12 +62,19 @@ class LocalBackend(
       + "is.hail.io.compress.BGzipCodecTbi,"
       + "org.apache.hadoop.io.compress.GzipCodec")
 
+  private[this] val flags = HailFeatureFlags.fromEnv()
   private[this] val theHailClassLoader = new HailClassLoader(getClass().getClassLoader())
 
+  def getFlag(name: String): String = flags.get(name)
```

Comment:
might discover this as I keep reading, but why do we need this if ExecuteContext has it?

Reply:
When in use, the LocalBackend and SparkBackend are the source of truth for flags. I use getFlag/setFlag on the backend to implement `hl._set_flag` and `hl._get_flag`. Put another way: the Python client might want to know a flag value when there is no execution occurring.